File: 01_Basico/03 - Lacos_Repeticao/Respsota_em_Python/questao19.py
grupoA = 0
grupoB = 0
grupoC = 0
grupoD = 0

while (True):
    numero = int(input("Para encerrar digite um número negaito\nDigite um número inteiro: "))

    if numero < 0:
        break
    elif numero > 0 and numero <= 25:
        grupoA += 1
    elif numero > 25 and numero <= 50:
        grupoB += 1
    elif numero > 50 and numero <= 75:
        grupoC += 1
    elif numero > 75 and numero <= 100:
        grupoD += 1
    else:
        print("Opção inválida o número deve ser um inteiro entre 0 e 100") 
    
print(f"Total de número que estão entre 0-25: {grupoA}")
print(f"Total de número que estão entre 26-50: {grupoB}")
print(f"Total de número que estão entre 51-75: {grupoC}")
print(f"Total de número que estão entre 76-100: {grupoD}")

File: 01_Basico/03 - Lacos_Repeticao/Resposta_em_Kotlin/questao19.kt
fun main(){
    var grupoA = 0
    var grupoB = 0
    var grupoC = 0
    var grupoD = 0
    var continuar = true
    while (continuar) {
        println("Para encerrar digite um número negaito\nDigite um número inteiro: ")
        var numero = readLine()!!.toInt()

    when (true) {
        (numero < 0) -> continuar = false
        (numero > 0 && numero <= 25) -> grupoA++
        (numero > 25 && numero <= 50) -> grupoB++
        (numero > 50 && numero <= 75) -> grupoC++
        (numero > 75 && numero <= 100) -> grupoD++
        else -> println("Opção inválida o número deve ser um inteiro entre 0 e 100") 
        }
    }
    println("Total de número que estão entre 0-25: $grupoA")
    println("Total de número que estão entre 26-50: $grupoB")
    println("Total de número que estão entre 51-75: $grupoC")
    println("Total de número que estão entre 76-100: $grupoD")
}